bootstrap/bootstrap.go

```diff
@@ -293,11 +293,56 @@ func (b *Bootstrap) executeHook(ctx context.Context, hookCfg HookConfig) error {
 
 	b.shell.Headerf("Running %s hook", hookName)
 
+	isBinary, err := shellscript.IsBinaryExecutable(hookCfg.Path)
+	if err != nil {
+		return fmt.Errorf("checking if %s hook is a binary: %w", hookName, err)
+	}
+
+	sb, err := shellscript.ShebangLine(hookCfg.Path)
```

Comment:
I think `sb` is a bit short/cryptic for the size of the scope that we're in here (I love short vars for short scopes), and alongside the longer-and-more-descriptive `isBinary`. Perhaps it could be called `shebang` instead, which is still short but with descriptiveness similar to `isBinary`, and also helps clarify that while `isBinary` is a boolean, this one is a string?